openforcefield/typing/engines/smirnoff/io.py

```diff
@@ -312,8 +303,6 @@ def prepend_all_keys(d, char='@'):
                     prepend_all_keys(item)
 
         prepend_all_keys(smirnoff_data['SMIRNOFF'])
-        print(smirnoff_data)
-        print()
         return xmltodict.unparse(smirnoff_data, pretty=True)
 
     def to_xml(self, smirnoff_data):
```

Comment:
Like you said, it would have to be `ff_object.get_io_handler('XML').to_xml()`. And all is does is call `XMLParameterIOHandler.to_string()` anyway, which is easier to access via `ff_object.to_string('XML')`

It's so unlikely that this would ever be used, that I'd be OK to remove it.